src/pages/policy/output/Analysis.jsx

```diff
@@ -59,14 +70,44 @@ export default function Prompt(props) {
   });
   const openai = new OpenAIApi(configuration);
   const [showPrompt, setShowPrompt] = useState(false);
-  const lines = analysisPrompt.split("\n");
+  const lines = prompt.split("\n");
+
+  const handleAudienceChange = (audienceValue) => {
+    setAudience(audienceValue);
+  };
+
+
+  const buttonWidth = "80px";
+  const activeColor = "#6c757d";
```

Comment:
If this is `DARK_GRAY` we have a `style.colors` constant holding it, would be good to link it up to that.